### DISCUSSION TITLE
Necessary Features missing.

### DISCUSSION THREAD

**Discussion by OmarShawky1** [alainm23/planify#668] [General]
I use Microsoft To-Do list because it is multi platform and it covers almost all what i need till the extent that i stopped using OneNote; it is not available on Linux by default and the only tweak is use a Web-App called Ao on Snap; without going into too much details. i cannot stop using this program and use yours (although it is native and actually pretty good as a UI even compared to theirs on Windows) because of those missing Important Features:

1-It is not available on mobile.
2-i cannot add any notes (Notes are very different from check list, please use To-Do to understand and investigate more)
3-I cannot attach any files (especially photos).

**Comment by BeatLink** [comment]
1. CalDAV synchronization is planned for upcoming planner versions. This means that in the future, you can connect planner with Nextcloud, Google Tasks and other online task platforms that use the CalDAV protocol. This means that any application, mobile, desktop or otherwise, will soon be compatible with planner so long as it uses the CalDAV protocol too!

2. As for taskless notes, seems like others would like to see it implemented too! This issue was opened 5 days ago here. https://github.com/alainm23/planner/issues/660. Head on over and show your support with a thumbs up.

3. As for attachments and images, this too seems to be another much requested feature. There are issues open for attachment support here https://github.com/alainm23/planner/issues/375 and image support here https://github.com/alainm23/planner/issues/488

**Reply by OmarShawky1** [reply]
Done, waiting for your amazing work :)

**Reply by yodatak** [reply]
Thanks for this awsome news did you intent to have nextcloud deck works with the cards from Planner ?
https://apps.nextcloud.com/apps/deck

**Reply by BeatLink** [reply]
> Done, waiting for your amazing work :)

I only help out @alainm23 with issue triaging and stuff. He deserves  *all* the credit!
